software/utils/debug/controller_debug.py

```diff
@@ -301,16 +297,17 @@ def test_run(self, name):
         print("\n")
         self.variables_set(test.scenario.ventilator_settings)
         self.variables_force_off()
+        self.trace_set_period(test.scenario.trace_period)
+        self.trace_select(test.scenario.trace_variable_names)
+        self.trace_start()
         time.sleep(test.scenario.capture_ignore_secs)
 
         print(
             f"\nStarting data capture for {test.scenario.capture_duration_secs} seconds"
         )
         self.trace_flush()
```

Comment:
In case the wait time is longer than the trace capacity, this will empty the trace buffer but not restart the trace, and we will end up with no data.

Reply:
Good catch. I guess it's better to just wait to start the trace after the wait.